File: Conditionals/time_notes.py
#Anthony Petersen, Time Of Day
import time

#first instance of time in programming
first_time  = time.gmtime()

#get time in seconds
current = time.time()
#print(seconds_time) #seconds since Janurary 1 1970

#current date and time the normal way
now = time.ctime(current)
#print(date)

#part of the time
local_time = time.localtime(current)
day = local_time.tm_wday
hour = local_time.tm_hour
minute = local_time.tm_min
seconds = local_time.tm_sec
print(day)

File: Conditionals/time_notes.c
// Anthony Petersen, Time Of Day
#include <stdio.h>
#include <time.h>


int main(void){
    time_t seconds; //time since January 1st 1970
    seconds = time(NULL);
    //printf("Time since January 1st 1970 in seconds\n=\n%d \n", seconds);

    //current time
    time_t rawtime;
    struct tm * timeinfo;
    time(&rawtime);
    timeinfo = localtime(&rawtime);
    //printf("Our current time and date is %s \n", asctime(timeinfo));

    //current hour
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d", hour);
    return 0;
}